pkg/schedule/checker/merge_checker_test.go

```diff
@@ -56,6 +56,7 @@ func TestMergeCheckerTestSuite(t *testing.T) {
 
 func (suite *mergeCheckerTestSuite) SetupTest() {
 	cfg := mockconfig.NewTestOptions()
+	gcInterval = 100 * time.Millisecond
```

Comment:
Need revert it

Reply:
`gcInterval` is 1 minute by default, which is too long for test.

Reply:
get